ghcup-prereleases-0.0.7.yaml

```diff
@@ -636,6 +636,93 @@ ghcupDownloads:
+        A_64:
+          Darwin:
+            unknown_versioning:
+              dlHash: 25c32514e6aba55c510f01bb264d959a56c5fed114d7ef508de665c5eeaf47ed
+              dlSubdir: ghc-9.6.0.20230210-x86_64-apple-darwin
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230210/ghc-9.6.0.20230210-x86_64-apple-darwin.tar.xz
+          Linux_Alpine:
+            unknown_versioning:
+              dlHash: 5722aca0e4da55e1e05c20bfcd63d21c59d0800e1850e87b4b9ad0066a918279
+              dlSubdir: ghc-9.6.0.20230210-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230210/ghc-9.6.0.20230210-x86_64-alpine3_12-linux-static-int_native.tar.xz
+          Linux_CentOS:
+            ( >= 7 && < 8 ): &ghc-961alpha3-64-centos7
+              dlHash: 0f68876231e7bab3f261238533461aed0e231fa93b1483eef08a91dc510eddc1
+              dlSubdir: ghc-9.6.0.20230210-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230210/ghc-9.6.0.20230210-x86_64-centos7-linux.tar.xz
+            unknown_versioning: *ghc-961alpha3-64-centos7
+          Linux_Debian:
+            (>= 10 && < 11): &ghc-961alpha3-64-deb10
+              dlHash: fa0c2b8e31ea182f5111feeec3c3fcff2074857b5a068d8902519fdcc96b26e3
+              dlSubdir: ghc-9.6.0.20230210-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230210/ghc-9.6.0.20230210-x86_64-deb10-linux.tar.xz
+            < 10: &ghc-961alpha3-64-deb9
+              dlHash: 9a43cad78680d6a47e338ae5bc13deada164d7203996f7791e6957945c5aa5f3
+              dlSubdir: ghc-9.6.0.20230210-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230210/ghc-9.6.0.20230210-x86_64-deb9-linux.tar.xz
+            '>= 11': &ghc-961alpha3-64-deb11
+              dlHash: d552b709426dd8e1afa499789bd9ac775c1f062005656b4c41d28bb338d6692e
+              dlSubdir: ghc-9.6.0.20230210-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230210/ghc-9.6.0.20230210-x86_64-deb11-linux.tar.xz
+            unknown_versioning: *ghc-961alpha3-64-deb11
+          Linux_Fedora:
+            '>= 33': &ghc-961alpha3-64-fedora33
+              dlHash: ade217760de958e6ed8693b16245bd244a2ccf54c262b37222d784b839d8a699
+              dlSubdir: ghc-9.6.0.20230210-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230210/ghc-9.6.0.20230210-x86_64-fedora33-linux.tar.xz
+            unknown_versioning: *ghc-961alpha3-64-centos7
+          Linux_Mint:
+            < 20: *ghc-961alpha3-64-deb9
+            '>= 20': *ghc-961alpha3-64-deb10
+          Linux_RedHat:
+            unknown_versioning: *ghc-961alpha3-64-centos7
+          Linux_Ubuntu:
+            ( >= 16 && < 19 ): *ghc-961alpha3-64-deb9
+            unknown_versioning: *ghc-961alpha3-64-deb10
+          Linux_UnknownLinux:
+            unknown_versioning: *ghc-961alpha3-64-fedora33
```

Comment:
As per the previous prerelease, I decided to use rocky linux for unknown linux, see the previous mapping

```yml
          Linux_UnknownLinux:
            unknown_versioning:
              dlHash: 4c9dba7e69708fa172e7b8bac38e671f58829d73a29e489bbd7f99ce6a136989
              dlSubdir: ghc-9.6.0.20230128-x86_64-unknown-linux
              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230128/ghc-9.6.0.20230128-x86_64-rocky8-linux.tar.xz
```

Reply:
Cheers.